ext/client.go

```diff
@@ -199,9 +199,12 @@ func (c *client) CreateContainer(hostname string, image_alias string, image_serv
 
 	form := url.Values{}
 	form.Set("container[hostname]", hostname)
-	form.Set("container[image_alias]", image_alias)
-	form.Set("container[image_server]", image_server)
-	form.Set("container[image_protocol]", image_protocol)
+	form.Set("container[source_type]", source_type)
+	form.Set("container[source_alias]", source_alias)
+	form.Set("container[source_certificate]", source_certificate)
+	form.Set("container[source_mode]", source_mode)
+	form.Set("container[remote_server]", remote_server)
```

Comment:
do we have these params (remote_server, remote_protocol) accepted on mono?
i thought it should be source[remote[name]]

Reply:
Done. please check Mas.